### PR TITLE
ESLint: Fix in loops and no process exit

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
Fix more ESLint errors!

Should be just two more PR's after this one!

### CODE REVIEW

File: web/scripts/generateZonesConfig.ts [hunk 1]
```diff
@@ -31,12 +31,14 @@ const mergeZones = () => {
...
-      point[1] = round(point[1], 4);
-    });
+    if (zoneConfig?.bounding_box) {
+      for (const point of zoneConfig.bounding_box) {
+        point[0] = round(point[0], 4);
+        point[1] = round(point[1], 4);
+      }
+    }
 
-    for (const key in zoneConfig) {
```
Comment (madsnedergaard) [inline web/scripts/generateZonesConfig.ts:39]: What's the rule for not allowing `x in y`?

File: web/.eslintrc.json [hunk 2]
```diff
@@ -169,7 +169,8 @@
         "project": ["./tsconfig.node.json"]
       },
       "rules": {
-        "unicorn/no-array-reduce": "off"
+        "unicorn/no-array-reduce": "off",
+        "unicorn/no-process-exit": "off"
```
Comment (madsnedergaard) [inline web/.eslintrc.json:173]: Thanks for turning these off :D

**Review by madsnedergaard — APPROVED** [review]
Not quite sure why `in` is not allowed, but otherwise LGTM

File: web/scripts/generateZonesConfig.ts (hunk 1 above)
Comment (VIKTORVAV99) [reply web/scripts/generateZonesConfig.ts:39]: Something about duplicating the underlying prototype objects and slowing down performance. So `Object.keys(object)` creates a new iterator and then of loops over it as if it was an array.

I didn't dig too much into it though and just fixed it 😅

**Pull request merged**